library/math.po

```diff
@@ -307,6 +314,9 @@ msgid ""
 "floor of the exact square root of *n*, or equivalently the greatest integer "
 "*a* such that *a*\\ ² |nbsp| ≤ |nbsp| *n*."
 msgstr ""
+"Renvoie la racine carrée entière du nombre positif *n*. C'est la partie "
+"entière de la valeur exacte de la racine carrée de *n*, ou, de manière "
```

Comment:
```suggestion
"entière de la valeur exacte de la racine carrée de *n* ou, de manière "
```
pas de virgule avant la conjonction de coordination.

Reply:
Noté.